Makefile

```diff
@@ -5,17 +5,23 @@ V := @
 BINDIR := ./bin
 SY_CRI := $(BINDIR)/sycri
 SY_CRI_SELINUX := $(BINDIR)/sycri-selinux
+SECCOMP := $(shell echo '\#include <seccomp.h>\nint main() { }' | gcc -x c -o /dev/null -lseccomp - >/dev/null 2>&1; echo $$?)
+BUILD_TAGS :=
+ifeq ($(SECCOMP), 0)
```

Comment:
I suggest to add a warning message if seccomp dependency is not installed

Reply:
will do